src/SortableContainer/index.js

```diff
@@ -636,9 +641,43 @@ export default function sortableContainer(
       const prevIndex = this.newIndex;
       this.newIndex = null;
 
+      // Returns whether a node location is a valid location for another node
+      // to be moved into
+      const isAllowedToMoveToNode = ({node}) => !node.sortableInfo.locked;
```

Comment:
If this becomes:

```
({node}) => !node.sortableInfo.locked && !node.sortableInfo.disabled
```

This performs the same positional locking for disabled items that is requested in: https://github.com/clauderic/react-sortable-hoc/issues/648